src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,29 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info(String.format("completeOrder method called for userId: %d", userId));
```

Comment:
Let's make our log messages informative. Instead of using String.format, use logger's built-in formatting capabilities. For example: logger.info("completeOrder method called for userId: {}", userId);